rbc/irtools.py

```diff
@@ -140,18 +167,17 @@ def init(self):
 # GPU Context classes
 
 
-class RemoteGPUTargetContext(cpu.CPUContext):
+class JITRemoteGPUTargetContext(cpu.CPUContext):
 
-    def __init__(self, typing_context, target_info):
-        self.target_info = target_info
+    def __init__(self, typing_context):
         super().__init__(typing_context)
 
     @compiler_lock.global_compiler_lock
     def init(self):
-        self.address_size = self.target_info.bits
+        target_info = TargetInfo()
```

Comment:
I guess this is not required anymore

Reply:
I am not sure what do you mean but I'd keep `is32bit` and `address_size` to correspond to the bitness of the target: some GPU cards are 32-bit while the server runs 64-bit linux.